draft-bbs-signatures.md

```diff
@@ -369,15 +369,11 @@ Outputs:
 
 Procedure:
 
-1. (W, H0, H) = octets_to_point(PK)
+1. W = octets_to_point(PK)
 
 2. If W == Identity_G2, return INVALID
 
-3. result = subgroup_check(W) && subgroup_check(H0)
-
-4. for i in 0 to len(H): result &= subgroup_check(H[i])
-
-5. return result
+3. return subgroup_check(W)
```

Comment:
Swap so we do the subgroup_check before checking the public key does not equal Identity_G2

Reply:
Addressed